Mathlib/Data/Real/ENNReal.lean

```diff
@@ -2298,6 +2301,15 @@ section infᵢ
 
 variable {ι : Sort _} {f g : ι → ℝ≥0∞}
 
+theorem toNNReal_infᵢ (hf : ∀ i, f i ≠ ∞) : (infᵢ f).toNNReal = ⨅ i, (f i).toNNReal := by
+  cases isEmpty_or_nonempty ι
+  · rw [infᵢ_of_empty, top_toNNReal, NNReal.infᵢ_empty]
+  · lift f to ι → ℝ≥0 using hf
+    simp only [← coe_infᵢ, toNNReal_coe]
+
+theorem toReal_infᵢ (hf : ∀ i, f i ≠ ∞) : (infᵢ f).toReal = ⨅ i, (f i).toReal := by
+  simp only [ENNReal.toReal, toNNReal_infᵢ hf, NNReal.coe_infᵢ]
```

Comment:
Am I right in saying I remember seeing these be backported?

Reply:
(if not, either we should backport them, or we should add a porting note

Reply:
leanprover-community/mathlib#18946